packages/labs/scoped-registry-mixin/src/scoped-registry-mixin.ts

```diff
@@ -48,10 +64,22 @@ export function ScopedRegistryHost<SuperClass extends LitElementConstructor>(
         );
       }
 
-      const renderRoot = (this.renderOptions.creationScope = this.attachShadow({
+      const renderRoot = this.attachShadow({
         ...shadowRootOptions,
         customElements: constructor.registry,
-      }));
+      });
+
+      this.renderOptions.creationScope = hasLegacyScopedCustomRegistry
+        ? renderRoot
+        : {
+            importNode(node: Node, deep?: boolean): Node {
+              return document.importNode(node, {
+                customElements: constructor.registry,
+                selfOnly: !deep,
+                deep /* Safari Tech preview <= 214 */,
```

Comment:
I don't think we should support a small, now obsolete, version range of Safari Tech Preview. Anyone who has this not work on STP can just update their STP.

Reply:
Removed.